packages/manager/modules/octavia-load-balancer/src/onboarding/controller.js

```diff
@@ -13,8 +26,13 @@ export default class OctaviaLoadBalancerOnboardingCtrl {
     }));
     this.illustration = illustration;
     this.cta = CTA[ovhSubsidiary] || CTA.DEFAULT;
+    this.privateNetworkHelpUrl =
+      PRIVATE_NETWORK_HELP[ovhSubsidiary] || PRIVATE_NETWORK_HELP.DEFAULT;
     this.atInternet = atInternet;
     this.ctaTrackName = `${TRACKING_CHAPTER_1}::${TRACKING_NAME}::add`;
+    this.$window = $window;
+    this.coreURLBuilder = coreURLBuilder;
+    this.$state = $state;
```

Comment:
```suggestion
```
move $state in the state resolve section see : https://github.com/ovh/manager/pull/10001/files#r1338238994